doc/api/buffer.md

```diff
@@ -1290,8 +1306,8 @@ console.log(buf.length);
 ### buf.readDoubleBE(offset[, noAssert])
 ### buf.readDoubleLE(offset[, noAssert])
 
-* `offset` {Number} `0 <= offset <= buf.length - 8`
-* `noAssert` {Boolean} Default: false
+* `offset` {Number} Where to start reading. Must satisfy: `0 <= offset <= buf.length - 8`
```

Comment:
I've changed all `Number` references to `Integer` where it made sense.

I'm not sure about the bracket notation, it might look too array-like?